src/components/Table.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+
+
+class Table extends React.Component {
+    constructor( props ) {
+        super( props )
+        this.state = {
+            translationData: []
+        };
+    }
+
+    renderTableData() {
+        if ( this.state.translationData.length !== 0 ) {
+            return this.state.translationData.map( ( translation, index ) => {
+
+                const [ chinese, pinyin, english, italian, arabic, serbian, croatian, russian, german, hebrew, french,
+                    hungarian, slovak, spanish, portugues, turkce, greek, romanian ]
+                    = translation // Destructuring.
+
+                return (
+                    <tr key={ index }>
+                        <td>{ chinese + " " + pinyin }</td>
+                        <td>{ english }</td>
+                        <td>{ italian }</td>
+                        <td>{ arabic }</td>
+                        <td>{ serbian }</td>
+                        <td>{ croatian }</td>
+                        <td>{ russian }</td>
+                        <td>{ german }</td>
+                        <td>{ hebrew }</td>
+                        <td>{ french }</td>
+                        <td>{ hungarian }</td>
+                        <td>{ slovak }</td>
+                        <td>{ spanish }</td>
+                        <td>{ portugues }</td>
+                        <td>{ turkce }</td>
+                        <td>{ greek }</td>
+                        <td>{ romanian }</td>
+                    </tr>
+                )
+            })
+        } else {
+            return (
+                <tr>
+                    <td>{"loading"}</td>
```

Comment:
can we have some space inside the curly brackets? so something like { "loading" }